src/front-door/azext_front_door/custom.py

```diff
@@ -668,16 +671,21 @@ def create_waf_policy(cmd, resource_group_name, policy_name,
 
 def update_waf_policy(instance, tags=None, mode=None, redirect_url=None,
                       custom_block_response_status_code=None, custom_block_response_body=None,
-                      disabled=False):
+                      disabled=False, request_body_check=None, sku=None):
+    from azext_front_door.vendored_sdks.models import SkuName
     with UpdateContext(instance) as c:
         c.update_param('tags', tags, True)
 
+    with UpdateContext(instance.sku) as c:
+        c.update_param('name', sku, SkuName.classic_azure_front_door)
+
     with UpdateContext(instance.policy_settings) as c:
         c.update_param('enabled_state', 'Enabled' if not disabled else 'Disabled', 'Disabled')
         c.update_param('mode', mode, False)
         c.update_param('redirect_url', redirect_url, None)
         c.update_param('custom_block_response_status_code', custom_block_response_status_code, None)
         c.update_param('custom_block_response_body', custom_block_response_body, None)
+        c.update_param('request_body_check', request_body_check, 'Disabled')
```

Comment:
If user already create a instance which request_body_check is enabled. And then user update that instance without request_body_check parameter. Should we set it to Disabled?

Reply:
same question for "enabled_state" and "mode" property

Reply:
I think we should keep the origin value, instead of Disable.

Reply:
I have changed request_body_check, mode, enabled_state and sku to keep origin value during update